docs/docusaurus.config.js

```diff
@@ -57,30 +71,48 @@ const config = {
     navbar: {
       title: 'GlueSQL',
       items: [
-        {
-          type: 'doc',
-          docId: 'getting-started/rust',
-          position: 'left',
-          label: 'Getting Started',
-        },
-        {
-          type: 'doc',
-          docId: 'sql-syntax/intro',
-          position: 'left',
-          label: 'SQL Syntax',
-        },
-        {
-          type: 'doc',
-          docId: 'ast-builder/intro',
-          position: 'left',
-          label: 'AST Builder',
-        },
-        {
-          type: 'doc',
-          docId: 'storages/intro',
-          position: 'left',
-          label: 'Storages',
-        },
+        ...(!isBlog ? [
```

Comment:
Unless there's a good reason not to, why not use a conditional like the one on L42?

Reply:
though there's no special reason for that but L42 is for making js object and here is for array so I just intended to make it to look more simple.